src/OrleansRuntime/Catalog/Catalog.cs

```diff
@@ -728,10 +734,22 @@ private void CreateGrainInstance(string grainTypeName, ActivationData data, stri
                 //for log-view grains, install log-view adaptor
                 else if (grain is ILogConsistentGrain)
```

Comment:
not sure I understand what you mean. The event store is accessed from JournaledGrain, which is a LogConsistentGrain.